superset-frontend/plugins/legacy-plugin-chart-heatmap/src/Heatmap.js

```diff
@@ -211,8 +211,7 @@ function Heatmap(element, props) {
     return d3.scale.ordinal().domain(domain).range(d3.range(domain.length));
   }
 
-  // eslint-disable-next-line no-param-reassign
-  element.innerHTML = '';
+  element.replaceChildren();
```

Comment:
The old way is fine, but this is less icky. I'm NOT sure why this wasn't called out by eslint in any of these plugins. But, this should work fine (let's test to make sure)